src/routes/page-routes/application.tsx

```diff
@@ -46,15 +51,24 @@ const applicationRoutes = [
         errorElement: <RouteErrorBoundry />,
         element: <ComponentListTab />,
       },
+      {
+        path: 'integrationtests/tabs/:integrationTestTab',
```

Comment:
no need to use `tabs` in the path
```suggestion
        path: 'integrationtests/:integrationTestTab',
```